feature/more/src/main/java/com/epmedu/animeal/more/account/AccountScreenUI.kt

```diff
@@ -0,0 +1,71 @@
+package com.epmedu.animeal.more.account
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.foundation.button.AnimealSecondaryButtonOutlined
+import com.epmedu.animeal.foundation.button.AnimealTextButtonWithIcon
+import com.epmedu.animeal.foundation.theme.AnimealTheme
+import com.epmedu.animeal.foundation.topbar.BackButton
+import com.epmedu.animeal.foundation.topbar.TopBar
+import com.epmedu.animeal.resources.R
+
+@Composable
+internal fun AccountScreenUI(
+    onBack: () -> Unit
+) {
+    Scaffold(
+        modifier = Modifier.fillMaxSize(),
+        topBar = {
+            TopBar(title = stringResource(id = R.string.page_account)) {
+                BackButton(onBack)
+            }
+        }
+    ) {
+        Column(modifier = Modifier.wrapContentHeight().padding(horizontal = 26.dp)) {
+            Row(modifier = Modifier.fillMaxWidth()) {
+                AnimealTextButtonWithIcon(
+                    modifier = Modifier.fillMaxWidth().height(48.dp),
+                    icon = painterResource(id = R.drawable.ic_delete),
+                    text = stringResource(id = R.string.account_delete),
+                    color = Color.Red,
+                    onClick = { /* TODO implement account deletion */ }
+                )
+            }
+        }
+
+        Column(
```

Comment:
you can implement screen in several ways

1. Make two columns with `weight` modifier
2. Make single column and add `Spacer` with `weight 1f` between two buttons